transport/src/main/java/io/netty/channel/DefaultMaxBytesRecvByteBufAllocator.java

```diff
@@ -29,11 +30,17 @@
     private volatile int maxBytesPerRead;
     private volatile int maxBytesPerIndividualRead;
 
-    private final class HandleImpl implements Handle {
+    private final class HandleImpl implements ExtendedHandle {
         private int individualReadMax;
         private int bytesToRead;
         private int lastBytesRead;
         private int attemptBytesRead;
+        private final UncheckedBooleanSupplier defaultMaybeMoreSupplier = new UncheckedBooleanSupplier() {
+            @Override
+            public boolean get() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) A getX() method which returns a boolean should be named isX() [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3ABooleanGetMethodName)